src/expr/impl/Cargo.toml

```diff
@@ -51,7 +51,7 @@ itertools = { workspace = true }
 jsonbb = { workspace = true }
 linkme = { version = "0.3", features = ["used_linker"] }
 md5 = "0.7"
-moka = { version = "0.12", features = ["sync"] }
+moka = { version = "0.12.8", features = ["sync"] }
```

Comment:
Any specific reason that we want to pin to a specific version here?

Reply:
Since we update the version of iceberg_rust in this pr, and it depends on moka ^0.12.8